plugin/src/main/java/org/opensearch/ml/task/MLTaskRunner.java

```diff
@@ -14,23 +14,25 @@
 
 import static org.opensearch.ml.stats.StatNames.ML_EXECUTING_TASK_COUNT;
 
+import org.opensearch.action.ActionListener;
 import org.opensearch.ml.model.MLTask;
 import org.opensearch.ml.model.MLTaskState;
 import org.opensearch.ml.stats.MLStats;
+import org.opensearch.transport.TransportService;
 
 /**
  * MLTaskRunner has common code for dispatching and running predict/training tasks.
  */
-public class MLTaskRunner {
+public abstract class MLTaskRunner<S, T> {
```

Comment:
minor. generic type parameters can also be part of javadoc.

Reply:
thanks, missed this comment before merging, will fix in next PR